app/models/duffel.rb

```diff
@@ -7,6 +7,28 @@ def initialize(contents)
 
   def add_item(item_id)
     contents[item_id.to_s] ||= 0
-    contents[item_id.to_s] += 0
+    contents[item_id.to_s] += 1
+  end
+
+  def total
+    contents.values.sum
+  end
+
+  def count_of(item_id)
+    contents[item_id.to_s]
+  end
+
+  def subtotal
+    item_prices = contents.map do |item_id, quantity|
+      item = Item.find(item_id)
+      item.price * quantity
+    end
+    item_prices.sum
+  end
+
+  def duffel_items
+    contents.map do |item_id, quantity|
```

Comment:
Unused block argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used.